source/includes/databases-collections/time-series.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require 'vendor/autoload.php'; 
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
```

Comment:
This example don't need the sample dataset. I suggest changing to the default URI.
```suggestion
$uri = getenv('MONGODB_URI') ?: 'mongodb://localhost:27017';
```